plugin.js

```diff
@@ -48,7 +48,12 @@ class Plugin {
   }
 
   static processExpScopePlugins(exp_info, hb, lab_data, options) {
-    const pluginConfig = require(Plugin.getConfigFileName(options.env));
+    let pluginConfig = require(Plugin.getConfigFileName(options.env));
+
+    if(!options.isValidate)
+    {
+      pluginConfig = pluginConfig.filter((p) => p.id !== "build-validation");

```

Comment:
Should the id change to tool-validation?

Reply:
Yeah, we have changed it to tool-validation